test/ComparisonTest.java

```diff
@@ -0,0 +1,90 @@
+package grakn.simulation.test;
+
+import grakn.simulation.db.common.action.Action;
+import grakn.simulation.db.common.agent.base.Agent;
+import grakn.simulation.db.grakn.driver.GraknDriver;
+import grakn.simulation.db.neo4j.driver.Neo4jDriver;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+
+import static grakn.simulation.test.SimulationsForComparison.graknSimulation;
```

Comment:
What if we rename `SimulationsForComparison` to `SimulationRuns`? It feels consistent because we have a `SimulationRunner` which produces a `SimulationRun`, right?

And then access the simulation for each database like this:

```
import static grakn.simulation.test.SimulationRuns.grakn;
import static grakn.simulation.test.SimulationRuns.neo4j;
```

Reply:
You raise an interesting point, but no the `SimulationRunner` (still looking for a better name/architecture there) is the overall entrypoint which runs creates a `Simulation` object and runs it; Whereas, `SimulationsForComparison` simply creates a `Simulation` object for grakn and for Neo4j. On those grounds I'm happy-ish with the current naming (but open to more suggestions , though we can certainly make it better by accessing them as:
```
import static grakn.simulation.test.SimulationsForComparison.grakn;
import static grakn.simulation.test.SimulationsForComparison.neo4j;
```
I'll change that now

Reply:
I see, ok. I've got no other suggestion for the time being.